Scheduler/Pages/AboutPage.xaml

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms" xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" x:Class="Scheduler.Pages.AboutPage">
+    <StackLayout>
```

Comment:
No need to be snippy, but in this case you created StackLayout only for one label.